docs/installation/other_inst_scenarios/install_epinio_on_rancher_desktop.md

```diff
@@ -62,6 +66,13 @@ epinio login -u admin https://epinio.127.0.0.1.sslip.io
 
 For RancherDesktop on Linux, in order to be able to open ports from `443` (and above), in order to access the URL set in `global.domain` (i.e. 127.0.0.1.sslip.io), you need to set the start port of the `unprivileged` list to a lower number:
 
+<!--
+TODO:Does this need to be 80 as per the RD installation docs?
```

Comment:
Looking through the history this part came from @nunix .

Reply:
Thanks.